lumigator/python/mzai/backend/backend/services/datasets.py

```diff
@@ -119,6 +121,7 @@ def _save_dataset_to_s3(self, temp_fname, record):
             # Upload to S3
             dataset_key = self._get_s3_key(record.id, record.filename)
             dataset_path = self._get_s3_path(dataset_key)
+            # Deprecated!!!
```

Comment:
should we still use if deprectated?

Reply:
Since the scope was just replacing the mocks, and I didn't know if we used any special features, I just left a note :-/ 
Shall we add it to https://github.com/mozilla-ai/lumigator/issues/473 instead?